projects/ion/src/lib/tab/tab.component.html

```diff
@@ -12,4 +12,11 @@
 >
   <ion-icon *ngIf="iconType" class="tab-icon" [type]="iconType"></ion-icon>
   {{ label || 'Configure uma label ' }}
+
+  <ion-badge
+    data-testid="badge-tab"
+    *ngIf="badge"
+    [value]="badge.value"
+    type="primary"
```

Comment:
```suggestion
    type="primary"
```
This is gonna be always static?

Reply:
> This is gonna be always static?

for now, yes.